classes/game.rb

```diff
@@ -0,0 +1,18 @@
+require_relative './item'
+
+class Game < Item
+  attr_accessor :name, :multiplayer, :last_played_at
+
+  def initialize(name:, multiplayer:, last_played_at:, **all)
+    super(**all)
+    @name = name
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
+
+  private
+
+  def can_be_archived?()
+    true if @last_played_at > 2
+  end
```

Comment:
- [ ] Let's follow the requirement by ensuring that `can_be_archived?` here returns true if parent's method returns true AND if last_played_at is older than 2 years. 

- [ ] Let's make `can_be_archived?()` to be `can_be_archived?` instead for better readabilty.